tests/pytest_tests/unit_tests/test_torch.py

```diff
@@ -105,3 +105,11 @@ def test_watch_bad_argument(mock_run):
         ValueError, match="log must be one of 'gradients', 'parameters', 'all', or None"
     ):
         run.watch(net, log="bad_argument")
+
+def test_log_uint8_image(mock_run, capsys):
```

Comment:
you don't seem to be using it here.
```suggestion
def test_log_uint8_image(mock_run):
```

Reply:
Sorry, we actually didn't mean to leave this test in at all. We've removed this test entirely and just have the test that lives in test_data_types.py

Reply:
@dmitryduev if you are ok without this check and just using our other test I'll mark this as resolved and merge now that we passed CI tests